Utils/contribution_sync/sync_contrib_base.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import requests
+from github import Github, enable_console_debug_logging
+from github.Repository import Repository
+from typing import List, Bool
+
+
+def get_master_commit_sha(repo: Repository) -> str:  # noqa: E999
+    '''Return the sha commit of the master branch
+
+    Args:
+        repo (Repository): The repository whose master branch will be queried
+
+    Returns:
+        (str): The commit sha of the master branch's HEAD
+    '''
+    branch_data = repo.get_branch('master')
+    commit_sha = branch_data.commit.sha
+    return commit_sha
+
+
+def get_branch_names_with_contrib(repo: Repository) -> List[str]:  # noqa: E999
+    '''Return the list of branches that have the prefix of "contrib/"
+
+    Args:
+        repo (Repository): The repository whose branches will be searched and listed
+
+    Returns:
+        (List[str]): List of branch names that have the "contrib/" prefix
+    '''
+    branch_names = []
+    for branch in repo.get_branches():
+        if branch.name.startswith('contrib/'):
+            branch_names.append(branch.name)
+    return branch_names
+
+
+def is_pr(branch_name: str) -> Bool:  # noqa: E999
+    '''Returns whether a branch has a PR for it
+
+    Args:
+        branch_name (str): The name of the branch to check
+
+    Returns:
+        Bool: True if there is a PR for the branch, False otherwise
+    '''
+    response = requests.get(
+        f"https://api.github.com/repos/demisto/content/pulls?head=demisto:{branch_name}",
+        verify=False, headers={'Accept': 'application/json'}
+    )
+    if response.status_code == 200 and len(response.json()) > 0:
```

Comment:
can we change the condition to `if response.status_code == 200 and response.content` ?